ClassLibrary1/MyList Class.cs

```
@@ -0,0 +1,76 @@
+﻿namespace MyList;
+public class MyList<T> {
+   public MyList () => mArray = new T[mCapacity];
+   public MyList (int capacity = 4) {
+      if (capacity < 0) throw new Exception ("cannot create a MyList with a negative value");
+      else mArray = new T[capacity];
+   }
+
+   public T this[int index] {
+      get {
+         if (index < 0 || index > mArray.Length) throw new Exception ("IndexOutOfRangeException");

```

It should throw the below exception:
`if (index < 0 || index > mArray.Length) throw new IndexOutOfRangeException (); `
Moreover, there should be a getter also for this indexer as mentioned in the requirements.